src/components/PageSettingsModal/PageSettingsModal.tsx

```diff
@@ -186,9 +187,15 @@ export const PageSettingsModal = ({
                   <FormLabel mb={0} textColor="text.label">
                     Meta Description
                   </FormLabel>
-                  <FormLabel.Description useMarkdown color="text.description">
-                    Description snippet shown in search results. [Learn
-                    more](https://go.gov.sg/isomer-meta)
+                  <FormLabel.Description color="text.description">
+                    {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
```

Comment:
also, are we using a different version of ts here? O.o somehow I get an error, i need to put the eslint and ts ignore above the `FormLabel.Desc` portion, if not locally I still get this error liao

Reply:
actually, rather than this, could we just do a

```
 <FormLabel>
                <Text textStyle="body-2" color="text.description">
                  This is the preview image that shows when your site url is
                  shared on social media or messaging channels.{" "}
                  <Link
                    isExternal
                    href="https://guide.isomer.gov.sg/guide/settings"
                  >
                    Learn more
                  </Link>
                </Text>
              </FormLabel>
```
That way we dont depend on the desc types at all

Reply:
resolved offline, want htis to be tied to `Description` so that styling changes propagate downwards